workers/unity/Assets/Playground/Scripts/MonoBehaviours/CubeSpawnerInputBehaviour.cs

```diff
@@ -42,26 +43,26 @@ private void OnDeleteSpawnedCubeResponse(CubeSpawner.DeleteSpawnedCube.ReceivedR
             }
         }
 
-        private void Update()
+        private async void Update()
         {
             if (Input.GetKeyDown(KeyCode.Space))
             {
-                SendSpawnCubeCommand();
+                await SendSpawnCubeCommand();
             }
 
             if (Input.GetKeyDown(KeyCode.Backspace))
             {
-                SendDeleteCubeCommand();
+                await SendDeleteCubeCommand();
             }
         }
 
-        private void SendSpawnCubeCommand()
+        private async Task SendSpawnCubeCommand()
```

Comment:
Related to the above I think we can leave this is async void. But here we aren't actually doing anything with the response so there isn't a reason to await. Could just add a debug log if we want this is an example.